temba/msgs/models.py

```diff
@@ -831,9 +835,21 @@ def process_message(cls, msg):
 
         cls.mark_handled(msg)
 
+        # Chatbase parameters to track logs
+        chatbase_not_handled = True
+
         # if this is an inbox message, increment our unread inbox count
         if msg.msg_type == INBOX:
             msg.org.increment_unread_msg_count(UNREAD_INBOX_MSGS)
+        elif msg.msg_type == FLOW:
+            chatbase_not_handled = False
+
+        # Sending data to Chatbase API
+        chatbase_credentials = msg.org.get_chatbase_credentials()
+        if chatbase_credentials:
+            cls.send_chatbase_log(chatbase_credentials.get(CHATBASE_API_KEY.lower()),
```

Comment:
This is still a bit more complex than it needs to be. How about instead of `get_chatbase_credentials` packing things into a dict, and then having to extract them here... just return a tuple of (key, version)